### PR TITLE
Inverted output in neopixelbus

### PR DESCRIPTION
**Pull request by voibit**
## Description:
Adds support for inverted output recently added in the neopixelbus library. 

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome-docs](https://github.com/esphome/esphome-docs) with documentation (if applicable):** esphome/esphome-docs#441

## Checklist:
  - [ ] The code change is tested and works locally.
 - Only tested with eps8266 DMA 
  - [ ] Tests have been added to verify that the new code works (under `tests/` folder).

If user exposed functionality or configuration variables are added/changed:
  - [x] Documentation added/updated in [esphome-docs](https://github.com/esphome/esphome-docs).

### CODE REVIEW

File: esphome/components/neopixelbus/light.py [hunk 1]
```diff
@@ -116,10 +116,16 @@ def validate_method_pin(value):
...
 }
 
-
 def format_method(config):
     variant = VARIANTS[config[CONF_VARIANT]]
     method = config[CONF_METHOD]
+
+    if config[CONF_INVERT] == True:
+        if method == 'ESP8266_DMA':
+            variant = 'Inverted'+variant;
```
Comment (OttoWinter) [inline esphome/components/neopixelbus/light.py]: See also lint checks, the code here should be reformatted

```suggestion
            variant = 'Inverted' + variant
```

**Review by OttoWinter — COMMENTED** [review]
The neopixelbus version needs to be increased too. Please tell me which version of neopixelbus this shipped with and I will create a release for the fork.

**Comment by voibit** [comment]
Thanks, It was added i version 2.5.3.

**Review by OttoWinter — APPROVED** [review]
Thanks!

**Pull request merged**